src/khoj/routers/api_chat.py

```diff
@@ -291,6 +308,14 @@ async def chat(
             response_obj = {"response": no_entries_found_format}
             return Response(content=json.dumps(response_obj), media_type="text/plain", status_code=200)
 
+    if conversation_commands == [ConversationCommand.Notes] and is_none_or_empty(compiled_references):
```

Comment:
Nice, were we previously not returning no notes found if chat query with `/notes` command returned no results? Do we differentiate between notes command auto-selected by Khoj vs by user? If Khoj has auto-selected just the `/notes` command and no notes are found should we not revert to general in that case instead of returning `No notes found"?

Reply:
I don't think we should just revert to default in that case because the tool selection process only selecting `notes` is a pretty high signal that that would be the only tool desired. Most likely, the user would be trying to extract something from their notes and not have anything relevant, so this is useful context to return back.